addons/docs/src/blocks/ArgsTable.tsx

```diff
@@ -45,42 +44,46 @@ type ArgsTableProps = BaseProps | OfProps | ComponentsProps | StoryProps;
 
 const useArgs = (
   storyId: string,
-  storyStore: StoryStore
+  context: DocsContextProps<any>
```

Comment:
Should this be `DocsContextProps<AnyFramework>` as below?

Reply:
should `TFramework` be `AnyFramework` by default? Would eliminate a lot of boilerplate